### ISSUE TITLE
No visible label provided for 'Field', 'Condition' and 'Value' combo box on 'Add Filter' pane.: A11y_.NetAspireDashboard_StructuredLog_Add Filter_LabelsorInstructions

### ISSUE THREAD

**Issue by PoojaNamde** [dotnet/aspire#3396]
"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier."

**Please do not close this bug. This bug should only be closed by Trusted Tester after verification.**

### Tags:
#A11yTCS; #A11yMAS; #A11ySev2; #FTP; #Win11; #ChromiumEdge; #FTP; #DesktopWeb; #.NET Core; #SH_.Net Aspire Dashboard_Web_Mar2024; #WCAG3.3.2; #LabelsorInstruction;

### Environment Details:
Application Name: .Net Aspire Dashboard
URL: https://localhost:19888/
Chrome Version 122.0.2365.80 (Official build) (64-bit)
Window Version: Win 11
OS Build: 22621.3085

### Repro Steps:

1. Hit the URL: https://localhost:15887/ and login with valid credentials.
2. TAB to "Structured Log" tab item and hit ENTER key.
3. TAB to "Filter" and hit ENTER key.
4. Observe that no visible label provided for 'Field', 'Condition' and 'Value' combo box on 'Add Filter' pane.

### Actual Results:
No visible label provided for 'Field', 'Condition' and 'Value' combo box on 'Add Filter' pane.

**Same issue repro on 'Structured log' page -> 'Select application' combo box.**

### Expected Results:
Visible label should be provided for 'Field', 'Condition' and 'Value' combo box on 'Add Filter' pane.

### User Impact:
Screen reader users will not be able to understand the purpose of the dropdown and will miss the information about it.

### Attachment:
![3396_A11y_ NetAspireDashboard_StructuredLog_Add Filter_LabelsorInstructions](https://github.com/dotnet/aspire/assets/90901519/27232e6b-3749-4dfa-a3e0-08e3b49c2261)

![StructuredLogs_SelectApplication_Label](https://github.com/dotnet/aspire/assets/90901519/3a41667a-ac76-4f1f-988d-bdfa63deae17)

**Comment by kvenkatrajan** [comment]
This needs to wait for responsive design - moving it to 8.2

**Comment by adamint** [comment]
#Closed
